example/neumann-reduced/heat_equation_reduced.py

```diff
@@ -139,9 +144,12 @@ def advance(self, coupling_output):
 # Let preCICE steer the coupled simulation
 tic = perf_counter()
 solution = model.solution_space.empty()
+# assemble the system operator for given mu to avoid re-assembly in each iteration
+assembled_model = model.with_(operator=model.operator.assemble(model.parameters.parse([1, 2])))   # fails to converge for mu=[0.1, 2]
 while dealii.is_coupling_ongoing():
     # Compute the solution of the time step
-    data = model.compute(solution=True, coupling_input=coupling_input)
+    print(assembled_model.operator)
```

Comment:
Do you want to keep this print statement or was it meant for developing purpose?